rtgui/filmnegative.cc

```diff
@@ -311,3 +328,31 @@ void FilmNegative::editToggled()
         unsubscribe();
     }
 }
+
+void FilmNegative::recalcCropMediansClicked()
+{
+    std::array<float, 3> newMedians = {1.f};
```

Comment:
You are aware that only the first element will be `1.f`?

Reply:
Oops... sorry, i guess i was coding while sleeping :-D  It would be initialized to the wrong value anyway, and i forgot another check. I'll fix it this evening as soon as i get back from work.